doc/content/htgr/generic-pbr/generic-pbr_model.md

```diff
@@ -0,0 +1,343 @@
+`elem_number_of_hs`, and `power_fraction` options, respectively. Note that
+because heat is generated only in the fuel region, the power fraction of the
+this region is set to the desired value with those of the moderator and reflector
+regions are set to zero. The types of reactivity feedback of each region is
+specified with the `pke_material_type` option. The fuel (Doppler) reactivity
+coefficients are provided with the `fuel_doppler_coef` option while the
+moderator/reflector reactivity coefficients are provided with the
+`moderator_reactivity_coefficients` option. The numbers of axial elements for
+point kinetics calculation in the fuel (Doppler) and moderator/reflector regions
+are provided with the `n_layers_doppler` and `n_layers_moderator` options,
+respectively. Note that the number of PKE axial elements must match the length
+of the provided reactivity coefficients.
+
+By default, the reactivity feedback in the heat structures of a core channel
+is calculated using the layered-average solid temperature. However, users can
+choose to calculate the reactivity feedback in the fuel region with the fuel
+kernel temperature instead. This can be done by setting
+
+```language=bash
+
+        fuel_kernel_temperature = true
+        use_kernel_temperature_doppler = true
+
+```
+
+Currently, the fuel kernel temperature is calculated using the analytical model
+implemented in the TINTE code [!citep](TINTE2010). Additional information is
+needed as,
+
+```language=bash
+
+        molar_mass_uranium = 0.238
+        molar_mass_kernel_material = 0.27
+        heavy_metal_loading = 0.009
+        local_power_fraction = 1.0
+        heat_capacity_kernel_material = 300.0
+        modified_heat_flux_resistance = 2.6e-6
+
+```
+
+Detailed descriptions of these parameters are available in the SAM Theory Manual
+[!citep](Hu2021) and the TINTE report [!citep](TINTE2010).
+
+## Postprocessors
+
+This block is used to specify the output variables written to a `csv` file that
+can be further processed in Excel or other processing tools such as Python and
+Matlab. For example, to output the maximum fuel temperature in `F-1`:
+
+!listing htgr/generic-pbr/pbr.i block=max_Tsolid_F1
+
+and the mass flow rate in `F-1`
+
+!listing htgr/generic-pbr/pbr.i block=TotalMassFlowRateInlet_1
+
+## Preconditioning
+
+This block describes the preconditioner used by the solver. New users can leave this block unchanged.
+
+!listing htgr/generic-pbr/pbr.i block=Preconditioning
+
+## Executioner
+
+This block describes the calculation process flow. The user can specify the
+start time, end time, time step size for the simulation. The user can also choose
+to use an adaptive time step size with the `type = IterationAdaptiveDT` option.
+Other inputs in this block include PETSc solver options, convergence tolerance,
+quadrature for elements, etc. which can be left unchanged.
+
+!listing htgr/generic-pbr/pbr.i block=Executioner
```

Comment:
we now recommend a "running the simulation" section with just:
`sam-opt -i input_file.i`

Reply:
I had this in the 'generic-pbr_results.md' file. But for completeness I added it here as well.